packages/vhd-format-lwt/vhd-format-lwt.0.9.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "dave@recoil.org"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+authors: [ "Dave Scott" "Jon Ludlam" ]
+homepage: "https://github.com/mirage/ocaml-vhd"
+bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
+dev-repo: "git://github.com/mirage/ocaml-vhd"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" ]]
```

Comment:
This should also include `"-p" name`. Otherwise you will run tests for all the packages in this tarball.